__tests__/app.test.js

```diff
@@ -87,16 +94,53 @@ describe("GET /api/reviews", () => {
     
 })
 
-describe("Responds with 404 when given a bad route", () => {
-    test("GET /not-a-route, returns 404", () => {
-        return request(app)
-        .get("/APIOOOOO")
-        .expect(404)
-    })
-    
+describe("Responds with 404 when given a bad route /api/not-a-route", () => {
     test("GET /api/not-route, returns 404", () => {
         return request(app)
         .get("/api/revieeewwssssss")
         .expect(404)
     })
+})
+
+describe("GET /api/reviews/:review_id", () => {
+    test("Responds with a review object, containing all its properties", () => {
+        return request(app)
+            .get('/api/reviews/2')
+            .expect(200)
+            .then(( {body} ) => {
+                const {review} = body;
+                expect(review).toBeInstanceOf(Object);
+                expect(review).toEqual(
+                    expect.objectContaining({
+                        owner: expect.any(String),
+                        title: expect.any(String),
+                        review_id: expect.any(Number),
+                        category: expect.any(String),
+                        review_img_url: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number),
+                        designer: expect.any(String),
+                        review_body: expect.any(String)
+                        })
```

Comment:
You can be more specific here, if you are going to reviews/2 then you should know exactly what review comes back. Instead of using expect.any(), insert the actual values here!